datasets/ru/acf_bribetakers/crawler.py

```diff
@@ -51,7 +42,9 @@ def parse_result(context: Context, row: Dict[str, Any]):
     for tl in transliterations.split("\n"):
         tl = h.remove_bracketed(tl)
         entity.add("alias", tl)
-    entity.add("birthDate", parse_date(dob), original_value=dob)
+    if dob and NO_YEAR.match(dob):
```

Comment:
this needs to be `not NO_YEAR.match`

Reply:
it matches values like `value=17.07.`, so I think it should be as it is